db_assessment/2_load_op.sh

```diff
@@ -0,0 +1,13 @@
+THISD=$(pwd)
+bq mk -d --data_location=${DSLOC} ${DSNAME}
+cd ${OP_WORKING_DIR}/..
+for COLID in $(ls -1 ${OP_LOG_DIR}/opdb*| rev | cut -d '.' -f 2 | rev | sort | uniq)
+do
+python3 ./db_assessment/optimusprime.py -sep "${COLSEP}" -dataset ${DSNAME} -fileslocation ${OP_LOG_DIR} -projectname ${PROJECTNAME} -collectionid ${COLID} | tee ${THISD}/opload-${DSNAME}-${COLID}.log
```

Comment:
This should use the update execution format mentioned in the docs.  `optimus-prime` command

Reply:
Done